app/bundles/FormBundle/Collector/AlreadyMappedFieldCollector.php

```diff
@@ -11,13 +11,12 @@
  */
 final class AlreadyMappedFieldCollector implements AlreadyMappedFieldCollectorInterface
 {
-    private const EXPIRATION_IN_SECONDS = 18000; // 5 hours
+    private const EXPIRATION_IN_SECONDS = 18000;
 
-    private CacheProviderInterface $cacheProvider;
-
-    public function __construct(CacheProviderInterface $cacheProvider)
-    {
-        $this->cacheProvider = $cacheProvider;
+    public function __construct(
+        // 5 hours
+        private CacheProviderInterface $cacheProvider
+    ) {
```

Comment:
In this case the comment even doesn't belong to the property but to the const above it.
```suggestion
    public function __construct(private CacheProviderInterface $cacheProvider) {
```